src/pkg/types/driver.go

```diff
@@ -21,6 +21,8 @@ type Container struct {
 	VolumesFrom []string // container (default rw), container:rw, or container:ro
 	EntryPoint  []string
 	Command     []string // overridden by Run()
+	WorkDir     *string
+	DependsOn   map[string]string // container name -> condition
```

Comment:
for now, could we simply detect dependencies based on the volumes?

Reply:
then, lets at least make an enum out of the condition string.

Reply:
Good point, added.